src/main/java/nextstep/security/authentication/manager/ProviderManager.java

```diff
@@ -0,0 +1,24 @@
+package nextstep.security.authentication.manager;
+
+import nextstep.security.authentication.Authentication;
+import nextstep.security.authentication.provider.AuthenticationProvider;
+import nextstep.security.exception.AuthenticationException;
+
+import java.util.List;
+
+public class ProviderManager implements AuthenticationManager {
+    private final List<AuthenticationProvider> providers;
+
+    public ProviderManager(List<AuthenticationProvider> providers) {
+        this.providers = providers;
+    }
+
+    @Override
+    public Authentication authenticate(Authentication authenticationToken) {
+        return providers.stream().filter(
+                provider -> provider.supports(authenticationToken.getClass())
+        ).findFirst().orElseThrow(
+                AuthenticationException::notSupported
+        ).authenticate(authenticationToken);
+    }
```

Comment:
소소하지만 다음과 같이 개행을 하면 좀더 가독성이 좋을 것 같아요 😄 
```suggestion
    @Override
    public Authentication authenticate(Authentication authenticationToken) {
        return providers.stream()
        .filter(provider -> provider.supports(authenticationToken.getClass()))
        .findFirst()
        .orElseThrow(AuthenticationException::notSupported)
        .authenticate(authenticationToken);
    }
```

Reply:
[58614f1](https://github.com/next-step/spring-security-authentication/pull/28/commits/58614f1d4af41421b2f16d76eb1323fbf34c3bf9) 커밋에서 indentation 스타일 변경 반영했습니다.